public/app/features/explore/RunButton.tsx

```diff
@@ -1,27 +1,10 @@
 import React from 'react';
 import { RefreshPicker, defaultIntervals } from '@grafana/ui';
 import { selectors } from '@grafana/e2e-selectors';
-import memoizeOne from 'memoize-one';
-import { css } from 'emotion';
-import classNames from 'classnames';
-
-import { ResponsiveButton } from './ResponsiveButton';
-
 import { getTimeSrv } from 'app/features/dashboard/services/TimeSrv';
 
-const getStyles = memoizeOne(() => {
-  return {
-    selectButtonOverride: css`
-      label: selectButtonOverride;
-      .select-button-value {
-        color: white !important;
-      }
-    `,
-  };
-});
-
 export type Props = {
-  splitted: boolean;
+  small?: boolean;
```

Comment:
```suggestion
  isSmall?: boolean;
```
To be consistent with `isLive` prop below.